src/test/java/com/example/java/test/junior/developer/category/CategoryServiceJunitTest.java

```diff
@@ -0,0 +1,77 @@
+package com.example.java.test.junior.developer.category;
+
+import com.example.java.test.junior.developer.dto.CategoryDto;
+import com.example.java.test.junior.developer.mapper.CategoryMapper;
+import com.example.java.test.junior.developer.model.Category;
+import com.example.java.test.junior.developer.repository.CategoryRepository;
+import com.example.java.test.junior.developer.service.CategoryService;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+public class CategoryServiceJunitTest {
+    private final CategoryRepository categoryRepository = Mockito.mock(CategoryRepository.class);
+    private final CategoryMapper categoryMapper = new CategoryMapper();
```

Comment:
The idea of unit test is that all dependency components are mocked. So CategoryMapper should be mocked too.